File: sample-java/src/main/java/root/MainActivity.java
package root;

import android.app.Activity;
import android.os.Bundle;
import android.util.Log;

public final class MainActivity extends Activity {
  @Override
  protected void onCreate(final Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    Log.d("FromSource", "savedInstanceState: " + savedInstanceState);
  }
}

File: sample-kotlin/src/main/java/root/MainActivity.kt
package root

import android.app.Activity
import android.os.Bundle
import android.util.Log

internal class MainActivity : Activity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    Log.d("FromSource", "savedInstanceState: $savedInstanceState")
  }
}
